kybra/__main__.py

```
@@ -118,7 +126,7 @@ def create_paths(args: Args) -> Paths:
     py_file_names_file_path = f"{canister_path}/py_file_names.csv"
 
     # This is the path to the developer's Candid file passed into python -m kybra from the dfx.json build command
-    did_path = args["did_path"]
+    did_path = os.environ.get("CANISTER_CANDID_PATH", "CANDID_PATH_NOT_FOUND")
```

Should we throw if this is not found?

---

Maybe, lets talk about it

---

Done